### ISSUE TITLE
Fix accessibility issues on the public view

### ISSUE THREAD

**Issue by rija** [gigascience/gigadb-website#1357]
On the public part of the web site:

* External party has audited the website and noticed accessibility issues.
* Internally we ran an audit with WebAIM's Wave extension for Firefox and noticed many errors

We need to fix the accessibility errors both those detected by auditing tools and those that automated tools cannot detect.

Our automated acceptance tests suite have non-existent coverage of this kind of issue.
We do not expect additional tests scenarios to be added to the existing PHP-based test suite
We do expect thorough manual testing
If you are familiar with UI test tools like Playwright or Cypress, feel free to use them though.

**Comment by luistoptal** [comment]
@rija do we have an a11y report from the 3p audit? Do we have a list of errors from the internal audit?

**Comment by luistoptal** [comment]
Related subtickets

Priority 3 (essential)

- https://github.com/gigascience/gigadb-website/issues/1361
- https://github.com/gigascience/gigadb-website/issues/1362
- https://github.com/gigascience/gigadb-website/issues/1363
- https://github.com/gigascience/gigadb-website/issues/1366
- https://github.com/gigascience/gigadb-website/issues/1367
- https://github.com/gigascience/gigadb-website/issues/1368
- https://github.com/gigascience/gigadb-website/issues/1369
- https://github.com/gigascience/gigadb-website/issues/1370
- https://github.com/gigascience/gigadb-website/issues/1371
- https://github.com/gigascience/gigadb-website/issues/1372
- https://github.com/gigascience/gigadb-website/issues/1373
- https://github.com/gigascience/gigadb-website/issues/1374
- https://github.com/gigascience/gigadb-website/issues/1375
- https://github.com/gigascience/gigadb-website/issues/1376
- https://github.com/gigascience/gigadb-website/issues/1377
- https://github.com/gigascience/gigadb-website/issues/1378
- https://github.com/gigascience/gigadb-website/issues/1384
- https://github.com/gigascience/gigadb-website/issues/1387
- https://github.com/gigascience/gigadb-website/issues/1390

Priority 2 (important)

- https://github.com/gigascience/gigadb-website/issues/1364
- https://github.com/gigascience/gigadb-website/issues/1365
- https://github.com/gigascience/gigadb-website/issues/1379
- https://github.com/gigascience/gigadb-website/issues/1380
- https://github.com/gigascience/gigadb-website/issues/1381
- https://github.com/gigascience/gigadb-website/issues/1382
- https://github.com/gigascience/gigadb-website/issues/1389
- https://github.com/gigascience/gigadb-website/issues/1432

Priority 1 (helpful)

- https://github.com/gigascience/gigadb-website/issues/1383
- https://github.com/gigascience/gigadb-website/issues/1385
- https://github.com/gigascience/gigadb-website/issues/1386
- https://github.com/gigascience/gigadb-website/issues/1388

**Comment by rija** [comment]
Hi @luistoptal, 
there is no easy retrievable report from the old audit, but it was solely based on the result of the wave tool from WebAIM:
http://wave.webaim.org/
We were using their web extension for Firefox, so it can be easily re-run

**Comment by luistoptal** [comment]
@rija That's fine, I run an audit with the same tool, thanks!

**Comment by rija** [comment]
this work is done
